### ISSUE TITLE
Android lolipop | Error in Success callbackId: EmailComposer1091862430 : TypeError: fn is not a function : cordova.js:312

### ISSUE THREAD

**Issue by Mickelle** [katzer/cordova-plugin-email-composer#210]
I am getting the following error on Android Lolipop and Marshmallow.
Plugin is working fine in KitKat

Error in Success callbackId: EmailComposer1091862430 : TypeError: fn is not a function : cordova.js:312

Cordova version 6.1.1
Android platform version 5.1.1

![screen shot 2016-05-20 at 18 19 45](https://cloud.githubusercontent.com/assets/4597918/15428285/09b9f1f0-1eb8-11e6-8315-0889d9d4a844.png)

**Comment by shamaleyte** [comment]
I am getting the same error for ;
Cordova 6.1.1 
Android 6.0.1

**Comment by dtruel** [comment]
I'm having this issue as well

**Comment by camtera** [comment]
Same issue.   Thank you @dtruel, reverting back to 0.8.2 resolved issued
```
cordova plugin rm cordova-plugin-email-composer
cordova plugin add https://github.com/katzer/cordova-plugin-email-composer.git#0.8.2
```

**Comment by brentbynum** [comment]
It looks like the problem is caused when you don't provide a callback to the "open" call.You can work around this error without reverting by adding a callback to your usage like so: 

`cordova.plugins.email.open({
    subject: 'yada yada yada',
    body: 'blah blah blah.'
}, function(){ });`

...notice the no-op function at the end there.

**Comment by xiaofanwu** [comment]
@brentbynum I tried your method, it still doesn't work. the email available shows false. any idea? Thanks!

**Comment by rob5408** [comment]
As this [comment](https://github.com/katzer/cordova-plugin-email-composer/issues/240#issuecomment-248591938) points out, this [fork](https://github.com/hypery2k/cordova-email-plugin/tree/v1.1.1) fixed it.

**Comment by bdedardel** [comment]
Hi,

For my part

- revert to 0.8.2 works fine

```
cordova plugin add https://github.com/katzer/cordova-plugin-email-composer.git#0.8.2
```

- adding callback hide the error, but plugin still does not work

```
cordova.plugins.email.open({ subject: 'yada yada yada', body: 'blah blah blah.' }, function(){ });
```

Cordova 6.3.1
Android 5.0.1

**Comment by youradds** [comment]
I'm getting this as well: 

> phonegap.js:314 Uncaught TypeError: fn is not a function
>     at onAvailable (email_composer.js:101)
>     at Object.callbackFromNative (phonegap.js:293)
>     at <anonymous>:1:9

The problem is that I don't seem to be allowed to go back to 0.8.2:

> Error - Plugin (or a specific plugin version) not found in external repo: cordova-plugin-email-composer@0.8.2": Fetching plugin "cordova-plugin-email-composer@0.8.2" via npm Failed to fetch plugin cordova-plugin-email-composer - You can fix this here

I'm calling it with:

`<gap:plugin name="cordova-plugin-email-composer" version="0.8.2" source="npm" />`

Any ideas? This has caused a fatal issue in my app :(

Thanks

Andy

**Comment by katzer** [comment]
That has been fixed.

**Comment by youradds** [comment]
@katzer - is this fixed in an official release? 0.8.2 still seems to be the most recent release for me?

**Comment by katzer** [comment]
Not an official release but on master branch. The release comes in the next days.

**Comment by youradds** [comment]
Ah ok cool - I look forward to it :) I've got 2 project that "broke" with this bug, so it'd be nice to get them fixed up.

**Comment by katzer** [comment]
@youradds I've released the version 0.8.4

**Comment by youradds** [comment]
Awesome, thanks :) I'll give that a whirl later and see how it goes.

Cheers

Andy
